### DISCUSSION TITLE
Request for Information on installing OKD 4.9 with IPI on oVirt

### DISCUSSION THREAD

**Discussion by markamber** [okd-project/okd#1058] [Q&A]
Comparing [4.9 install docs](https://docs.okd.io/4.9/installing/index.html) and [4.8 install docs](https://docs.okd.io/4.8/installing/index.html) I notice that oVirt related install instructions are not listed in the sidebar for 4.9. My first question was going to be "should I just follow 4.8 docs to install 4.9 for oVirt IPI?" but after digging in I have found some breadcrumbs.

I have looked at #783 and it is now stale, but it seems to suggest some issues related to oVirt/Vsphere and certain versions of FCOS. 

I'm looking for table scraps here and I am coming up empty, the install overview page that oVirt is supported on both 4.8 and 4.9, but there are only instructions for 4.8. I just stood up a lab and I am ready to install 4.9 on oVirt and will probably try today perhaps, but it seems like something is up and I am not sure what and now I anticipate to run into issues that may be hard to pin to known bugs vs. first time user woes.

**Comment by sandrobonazzola** [comment]
Hi, I still see 4.9 instructions for oVirt in 4.9 docs here: https://docs.okd.io/4.9/installing/installing_rhv/installing-rhv-default.html and it's still in the side bar:
![image](https://user-images.githubusercontent.com/937860/149972689-8a1e3830-d37b-4dea-a6dd-727e5b628a7a.png)
